packages/ckeditor5-link/src/linkui.js

```diff
@@ -700,3 +711,28 @@ export default class LinkUI extends Plugin {
 function findLinkElementAncestor( position ) {
 	return position.getAncestors().find( ancestor => isLinkElement( ancestor ) );
 }
+
+// Returns next valid range for the fake visual selection marker.
+//
+// @private
+// @param {module:engine/model/range~Range} range Current range
+// @param {module:engine/model/documentselection~DocumentSelection#focus} focus Selection focus
+// @param {module:engine/model/writer~Writer} writer Writer
+// @returns {module:engine/model/range~Range|false} New valid range for the fake visual selection marker
```

Comment:
Where the function can return `false`?

Reply:
Good catch, I forgot to update the description... 👍